src/convertToObject.js

```diff
@@ -1,18 +1,23 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes string with styles
- * and returns object where CSS properties are keys
- * and values are the values of related CSS properties
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.split(';\n');
+  const sourceObj = {};
+
+  sourceArr.map(property => {
+    if (property !== '') {
+      const propArray = property.split(':');
```

Comment:
```suggestion
      const [key, value] = property.split(':');
```
 Можно сразу доставать из массива первый и второ элемент